Source/Tests/Paths/Setup/PathsFixture.cs

```diff
@@ -0,0 +1,23 @@
+namespace Bloodmasters.Tests.Paths.Setup;
+
+public class PathsFixture : IDisposable
+{
+    public static IEnumerable<object[]> GetAllDirPaths()
+    {
+        yield return new object[] { CodeImp.Bloodmasters.Paths.TempDir };
+        yield return new object[] { CodeImp.Bloodmasters.Paths.DownloadedResourceDir };
+        yield return new object[] { CodeImp.Bloodmasters.Paths.ConfigDirPath };
+        yield return new object[] { CodeImp.Bloodmasters.Paths.ScreenshotsDir };
+    }
+
+    public void Dispose()
+    {
+        if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("CI")))
```

Comment:
Is it ok?

Reply:
Why, though? I think that it's okay to clean this up on CI as well.

Reply:
It does clean on CI, and does not on local test runs (so local development files stay untouched)

Reply:
Actually you know what? I don't think we should clean these up at all.